test/kpod_logs.bats

```diff
@@ -8,6 +8,10 @@ function teardown() {
     cleanup_test
 }
 
+function setup() {
+    copy_images
```

Comment:
The `container_config.json` file specifies `redis:alpine`, which we don't pre-pull, so I don't think this helps us in this file.

Reply:
many of these are currently disabled. i only added the function so when they fix it, we can take advantage of them